### ISSUE TITLE
BUG - sev-3 - All - Missing TTY for military service calls to DMDC

### ISSUE THREAD

**Issue by g-opzt** [department-of-veterans-affairs/va-mobile-app#6965]
## What happened?

We are missing a TTY link to go with the phone number for Defense Manpower Data Center (DMDC) on the large panel that opens when a user clicks "What if my military service information doesn't look right?"

## Acceptance Criteria

- [ ] Locate correct number
- [ ] Add link in line with other phone number/TTY combinations in the app

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low, but particularly an accessibility issue 
- Frequency: Low. 80.7K users viewed the MilitaryInformation screen in the last 28 days. We don't have analytics currently on how many opened the large panel where this link is found.

- 3 - Low

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by htcollier** [comment]
I do not, but @mistymg may know the TTY number that we are supposed to use here.
